src/client/content/v1/images/svg/sprites.svg

```diff
@@ -4,7 +4,11 @@
 <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="176" height="32" viewBox="0 0 176 32">
 <defs>
 <g id="icon-fb">
+<<<<<<< HEAD
```

Comment:
These are GIT merge markings.  Please clean them up. In general, please check the files before checking them in.